testapi.pm

```diff
@@ -875,12 +875,25 @@ sub x11_start_program {    # no:style:signatures
     return $distri->x11_start_program($program, @args);
 }
 
+sub _croak ($command, $error) {
```

Comment:
I wonder if it would be better to move those helper functions outside the test API so that it stays more clean as the real outside interface. I mean just move it to a separate file, WDYT?